docs/calendar-draft.md

```diff
@@ -43,6 +35,11 @@ class MyCalendar {
 		isoDate: Temporal.Date
 	) : Temporal.Date;
 
+	/** Constructs a Temporal.Date from a free-form option bag */
+	fromFields(
```

Comment:
Minor bikeshedding, but IMO invoking this as `someCalendar.fromFields(someFields)` doesn't sound like what this method is described to do. Maybe `dateFromFields()` would be clearer?

Reply:
Done.  Also added an example for how this method would be called from `Temporal.Date.from`.